ios/Podfile.lock

```diff
@@ -722,4 +722,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 10567d1e173aae9ab5e898db3c41267100a01a22
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.2
```

Comment:
could you upgrade pod on your computer to 1.9.3? we are all the time changing it from 1.9.2 to 1.9.3 and from 1.9.3 to 1.9.2 :D